synapse/replication/tcp/redis.py

```diff
@@ -206,65 +250,62 @@ def __init__(
         self, hs: "HomeServer", outbound_redis_connection: txredisapi.RedisProtocol
     ):
 
-        super().__init__()
-
-        # This sets the password on the RedisFactory base class (as
-        # SubscriberFactory constructor doesn't pass it through).
-        self.password = hs.config.redis.redis_password
+        super().__init__(
+            hs,
+            uuid="subscriber",
```

Comment:
I think previously we defaulted this to `None`? Not sure if this change matters much?

Reply:
Yeah, I don't think it matters but it was annoying me that they want a string and we were passing in a None.

Reply:
Fair enough! 👍  I wasn't sure if this truly need to be unique is the only reason I raised an eyebrow.

Reply:
I think it only needs to be unique for concurrent connections, and we'll only ever have one subscriber connection :+1: